libs/client/src/service_account.rs

```diff
@@ -158,6 +161,7 @@ impl KanidmClient {
             .await
     }
 
+    // TODO: test coverage for this, but there's a weird issue with ACPs on apply
```

Comment:
This function will probably will go away in future
it was for a migration a few versions back and shouldn't be needed now.